### PR TITLE
PW-30 Write down code review methodology

### PR DESCRIPTION
**Pull request by MCFreddie777**

### CODE REVIEW

File: src/metodiky/metodika-code-review.tex [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    \subsection*{Obsah PR}
+
+    PR Obsahuje iba tie zmeny, ktoré sú relevantné s JIRA úlohou, ktorej sa daný PR týka.\\
+
+    \noindent Pull request obsahuje primárne zdrojové kódy. V prípade, že autor potrebuje na vzdialený server nahrať aj väčšie
+    súbory (rozumej 1MB a viac), použije technológiu \href{https://git-lfs.github.com/}{Git LFS (Large File Storage)}.\\
+
+    \noindent Ak je to nutné, pull request obsahuje aj dokumentáciu k danej zmene vo formáte \href{https://www.markdownguide.org/}{MarkDown}.\\
+
+    \noindent V prípade, že je to špecifikované v rámci JIRA úlohy, merge request obsahuje zmeny otestované spolu testami (Unit, E2E).
```
Comment (MaliniakovaKarin) [inline src/metodiky/metodika-code-review.tex]: How did you mean this sentence? I didn't understand what you wanted to say.

Comment (MCFreddie777) [reply src/metodiky/metodika-code-review.tex]: I meant it, that if the specification (acceptance criteria) of the JIRA task contain that this task is required to be implemented along with tests, the tests are the part of the Pull request.

Comment (MCFreddie777) [reply src/metodiky/metodika-code-review.tex]: I can reformat it, you're right, the sentence sounds weird.

**Review by MaliniakovaKarin — APPROVED** [review]
Now, it's okey.

**Pull request merged**